### PR TITLE
feat(ecs): support to bind EIP when creating an ECS instance

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:
API documentation:
https://support.huaweicloud.com/api-ecs/zh-cn_topic_0167957246.html#ZH-CN_TOPIC_0167957246__section1846944811410

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccComputeInstance_basic
=== PAUSE TestAccComputeInstance_basic
=== CONT  TestAccComputeInstance_basic
--- PASS: TestAccComputeInstance_basic (211.68s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       211.786s

$ make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeInstance_prePaid'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeInstance_prePaid -timeout 360m -parallel 4
=== RUN   TestAccComputeInstance_prePaid
=== PAUSE TestAccComputeInstance_prePaid
=== CONT  TestAccComputeInstance_prePaid
--- PASS: TestAccComputeInstance_prePaid (182.00s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       182.080s
```

### CODE REVIEW

File: docs/resources/compute_instance.md [hunk 1]
```diff
@@ -262,6 +271,9 @@ The following arguments are supported:
   Defaults to *false*. This parameter is valid if `charging_mode` is set to *postPaid*, and all data disks will be deleted
   in *prePaid* charging mode.
 
+* `delete_eip_on_termination` - (Optional, Bool) Specifies whether the EIP is released when the instance is terminated.
```
Comment (niuzhenguo) [inline docs/resources/compute_instance.md:274]: should eip associated with existing eip_id also be released? seems the codes below doesn't do this.

Comment (ShiChangkuo) [reply docs/resources/compute_instance.md:274]: as we discussed, the EIP will be released by the result of `shouldUnsubscribeEIP` function in pre-paid charging mode

Comment (niuzhenguo) [reply docs/resources/compute_instance.md:274]: fair enough

**Pull request merged**